packages/agreement-process/test/agreementConsumerDocuments.test.ts

```diff
@@ -49,40 +50,51 @@ import {
 
 describe("agreement consumer document", () => {
   describe("get", () => {
-    let agreement1: Agreement;
-
-    beforeEach(async () => {
-      agreement1 = {
+    it("should succed when the requester is the consumer or producer", async () => {
+      const agreement: Agreement = {
         ...getMockAgreement(),
         consumerDocuments: [
           generateMock(AgreementDocument),
           generateMock(AgreementDocument),
         ],
       };
 
-      await addOneAgreement(agreement1);
-      await addOneAgreement(getMockAgreement());
-    });
-
-    it("should succed when the requester is the consumer or producer", async () => {
-      const authData = getRandomAuthData(
-        randomArrayItem([agreement1.consumerId, agreement1.producerId])
-      );
```

Comment:
've added explicit tests for both of those cases, I think it's better than relying on a random pick in case something breaks, for example, only for the producer and the test picks the consumer N times in a row.